### PR TITLE
Delete the useless return

### PR DESCRIPTION
**Pull request by YuxiJin-tobeyjin**
**What this PR does / why we need it**:
Delete the useless return, cause err will always be set to nil.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
 `NONE`
**Special notes for your reviewer**:
 `NONE`
**Release note**:
 `NONE`
```release-note
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go [hunk 1]
```diff
@@ -54,15 +54,13 @@ func NewServerRunOptions() *ServerRunOptions {
 }
 
 // ApplyOptions applies the run options to the method receiver and returns self
```
Comment (gyliu513) [inline staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go]: s/ApplyOptions/ApplyTo

File: staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go [hunk 2]
```diff
@@ -54,15 +54,13 @@ func NewServerRunOptions() *ServerRunOptions {
...
-func (s *ServerRunOptions) ApplyTo(c *server.Config) error {
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go]: I'd like to keep this for congruence.  Without it, the code looks bugged with respect to all of its peers.

Comment (YuxiJin-tobeyjin) [inline staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go]: I see,thanks.I will close it.

**Pull request closed without merging**